### ISSUE TITLE
Selection Auto Scroll Defects

### ISSUE THREAD

**Issue by AntonSchmied** [nonlinear-labs-dev/C15#2077]
1. **Selection Auto Scroll is set to "Parameter" and Params are selected with the Panel Unit**: 
The focus is not set fully on the parameter. It remains cut off on the screen, if the frame of the module is visible on the screen. 

2. **Selection Auto Scroll is set to "Parameter and Preset" or "Preset" and Presets are selected with the Panel Unit**:
When selecting a preset, the focus is not set on the selected preset. Even if the preset is off screen in another bank, the auto scroll does not follow the selection.

**Comment by MatthiasSeeber** [comment]
I tested twice via IDE and on the C15 device.

On both, Auto Scroll on Parameter selection seems to work reliably, the selected parameter will become visible - although it may be partly obscured by the main Manu button, Undo/Redo or the Tab selector buttons (Sound, Preset, Parameter). Maybe, a little extra padding could solve this.

However, Auto Scroll on Preset doesn't really work. From the IDE, I observed that scrolling only fits the horizontal Bank position well, but seems to stick to the Bank's bottom border. (So, selecting for example the first preset of a large Bank, will make the Bank visible, but showing the last presets of that Bank, while the selected preset remains invisible)
On the Device, there doesn't seem to happen anything.
